accelerator/.config/ALZ-Powershell-Auto.config.json

```diff
@@ -284,6 +298,35 @@
           ],
           "validation": "azure_location"
         },
+        "SecondaryLocation":{
+          "source": "input",
+          "type": "string",
+          "description": "Secondary deployment location. (e.g 'westeurope')",
+          "Value": "",
+          "Targets": [
+            {
+              "Name": "parSecondaryLocation.value",
+              "Destination": "Parameters"
+            },
+            {
+              "Name": "parAutomationAccountLocation.value",
+              "Destination": "Parameters"
+            },
+            {
+              "Name": "parPolicyAssignmentParameters.value.ascExportResourceGroupLocation.value",
+              "Destination": "Parameters"
+            },
+            {
+              "Name": "parVirtualWanHubs.value[0].parHubLocation",
+              "Destination": "Parameters"
+            },
```

Comment:
I think these ones need to be removed or updated?

Reply:
Fixed